draft-meunier-web-bot-auth-architecture.md

```diff
@@ -190,31 +202,31 @@ Agents SHOULD extend `@signature-parameters` defined in {{generating-http-messag
 This `nonce` MUST be unique for the validity window of the signature, as defined by created and expires attributes.
 Because the `nonce` is controlled by the client, the origin needs to maintain a list of all nonces that it has seen that are still in the validity window of the signature.
 
-### Sending a request
+### Sending a request (#sending-request)
 
 An Agent SHOULD send a request with the signature generated above. Updating the architecture diagram, the flow look as follow.
 
 ~~~aasvg
-+---------+                                                                     +----------+
-|         |                              Exchange                               |          |
-|         |<=========================  Cryptographic  =========================>|          |
-|         |                              material                               |          |
-|  Agent  |                                                                     |  Origin  |
-|         |     .------------------------------------------------------.        |          |
-|         +-----| GET /path/to/resource                                |------->|          |
-|         |     | Signature: abc==                                     |        |          |
-+---------+     | Signature-Input: sig=(@authority);tag="web-bot-auth" |        +----------+
-                | Signature-Agent: signer.example.com                  |
-                '------------------------------------------------------'
++---------+                                                                    +----------+
+|         |                              Exchange                              |          |
+|         |<=========================  Cryptographic  ========================>|          |
+|         |                              material                              |          |
+|  Agent  |                                                                    |  Origin  |
+|         |     .-----------------------------------------------------.        |          |
+|         +-----| GET /path/to/resource                               |------->|          |
+|         |     | Signature: abc==                                    |        |          |
++---------+     | Signature-Input: sig=(@authority signature-agent);\ |        +----------+
```

Comment:
(this is also missing `keyid`)

Reply:
and expires, and created. I wanted this to be illustrative, but agree that fixing would be great. doing that in the next commit